guides/source/active_record_querying.md

```diff
@@ -2256,6 +2256,24 @@ irb> assoc.unscope(:includes).pluck(:id)
 
 [`pluck`]: https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pluck
 
+### `pick`
+
+[`pick`][] can be used to query single or multiple columns from the underlying table of a model. It accepts a list of column names as an argument and returns the first one of the specified column values ​​with corresponding data type.
+
+`pick` is a short-hand for `relation.limit(1).pluck(*column_names).first`, which makes it possible to replace code like:
+
+```ruby
+Customer.limit(1).pluck(:id).first
+```
+
+with:
+
+```ruby
+Customer.pick(:id)
+```
```

Comment:
The example shows how easy it is to shorten the code with pick.
But I think it misses the nuance that pick "is primarily useful when you have a relation that’s already narrowed down to a single row.".

````suggestion
```ruby
Customer.where(id: 1).pluck(:id).first
```

with:

```ruby
Customer.where(id: 1).pick(:id)
```
````

Reply:
Indeed, an important nuance was missing. I will reflect the suggested source code.

Reply:
Supported by https://github.com/rails/rails/pull/48404/commits/9904db03670c49d3f75f21d6cb72f832974475a9